### PR TITLE
DOCS: Document ALTER TABLE column encodings feature

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

**Comment by divyeshddv** [comment]
On a quick find, I also saw some usages of ENCODINGS with (storage_parameters/ storage_directives) in gpdb-doc/markdown/admin_guide/ddl/ddl-storage.html.md
Are these generated files that would get edited too? if not, can we also please include it in this PR

**Comment by VMWareJennyBeth** [comment]
@divyeshddv Thanks for catching this. I will update the PR to include those changes.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 1]
```diff
@@ -141,13 +142,15 @@ and subpartition\_element is:
...
 
-where column_storage_parameter is:
+where storage_directive is:
 
 ```
    blocksize={8192-2097152}
    compresstype={ZLIB|ZSTD|QUICKLZ|RLE_TYPE|NONE}
    compresslevel={0-9}
 ```
+where
```
Comment (divyeshddv) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:152]: Seems some editing error

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md [hunk 2]
```diff
@@ -241,7 +244,7 @@ Although you can specify the table's access method using the <code>appendoptimiz
 -   **SET SCHEMA** — Moves the table into another schema. Associated indexes, constraints, and sequences owned by table columns are moved as well.
 -   **ALTER PARTITION \| DROP PARTITION \| RENAME PARTITION \| TRUNCATE PARTITION \| ADD PARTITION \| SPLIT PARTITION \| EXCHANGE PARTITION \| SET SUBPARTITION TEMPLATE**— Changes the structure of a partitioned table. In most cases, you must go through the parent table to alter one of its child table partitions.
 
-**Note:** If you add a partition to a table that has subpartition encodings, the new partition inherits the storage directives for the subpartitions. For more information about the precedence of compression settings, see [Using Compression](../../admin_guide/ddl/ddl-storage.html#topic40).
+**Note:** If you add a partition to a table that has subpartition encodings, the new partition inherits the AOCO column encodings for the subpartitions. For more information about the precedence of compression settings, see [Using Compression](../../admin_guide/ddl/ddl-storage.html#topic40).
```
Comment (divyeshddv) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_TABLE.html.md:247]: Maybe we don't change this and keep this same also

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md [hunk 3]
```diff
@@ -266,13 +266,17 @@ COLLATE collation
 DEFAULT default\_expr
 :   The `DEFAULT` clause assigns a default data value for the column whose column definition it appears within. The value is any variable-free expression \(subqueries and cross-references to other columns in the current table are not allowed\). The data type of the default expression must match the data type of the column. The default expression will be used in any insert operation that does not specify a value for the column. If there is no default for a column, then the default is null.
 
-ENCODING \( storage\_directive \[, ...\] \)
+ENCODING \( storage_directive \[, ...\] \)
```
Comment (divyeshddv) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_TABLE.html.md:269]: missed \

**Review by divyeshddv — COMMENTED** [review]
Mostly looks good, some minor comments

**Review by divyeshddv — APPROVED** [review]
LGTM!

**Review by dyozie — APPROVED** [review]

**Pull request merged**